src/DetectChanges/BCBreak/ClassBased/ConstantRemoved.php

```diff
@@ -4,38 +4,35 @@
 
 namespace Roave\BackwardCompatibility\DetectChanges\BCBreak\ClassBased;
 
+use Psl\Dict;
+use Psl\Str;
+use Psl\Vec;
 use Roave\BackwardCompatibility\Change;
 use Roave\BackwardCompatibility\Changes;
 use Roave\BetterReflection\Reflection\ReflectionClass;
 use Roave\BetterReflection\Reflection\ReflectionClassConstant;
 
-use function array_diff_key;
-use function array_filter;
-use function array_map;
-use function array_values;
-use function Safe\sprintf;
-
 final class ConstantRemoved implements ClassBased
 {
     public function __invoke(ReflectionClass $fromClass, ReflectionClass $toClass): Changes
     {
-        $removedConstants = array_diff_key(
+        $removedConstants = Dict\diff_by_key(
```

Comment:
Similar to other comments about `array_keys()` and `array_intersect_key()`, I would suggest keeping this as it was